Sources/PriorityQueueModule/PriorityQueue.swift

```diff
@@ -361,3 +361,45 @@ extension PriorityQueue {
         }
     }
 }
+
+extension PriorityQueue {
+
+    /// Adds sequence conformance to both the min and max view of the priority queue
+
+    public struct Iterator: Sequence, IteratorProtocol {
```

Comment:
Let's move this out into a separate file `PriorityQueue+Sequence.swift`.

Reply:
Should we still do that even though the PriorityQueue doesn't **directly** conform to the Sequence protocol? It just provides access to an iterator (which does conform). 

I'm thinking we can place it in a file called `PriorityQueue+Iterator.swift`, as that is more descriptive as to what is exactly going on.

Reply:
Either way works for me. I agree that it could be confusing to call it `+Sequence`, because it doesn't conform directly. If you want to call it `+Iterator` instead I'm fine with that.